EIPS/eip-7577.md

```diff
@@ -0,0 +1,107 @@
+---
+eip: 7577
+title: Versioning Scheme for EIPs
+description: Use a versioning scheme for EIPs based on changes made to their Specification section.
+author: danceratopz (@danceratopz), Ahmad Bitar (@smartprogrammer93)
+discussions-to: https://ethereum-magicians.org/t/add-eip-versioning-scheme-for-eips/17295
+status: Draft
+type: Meta
+created: 2023-12-13
+---
+
+## Abstract
+
+This EIP introduces a versioning scheme for [Standards Track](./eip-1.md#eip-types) EIPs by applying semantic versioning based on changes made to the EIP's Specification section once its status has changed from `Draft` to `Review`.
```

Comment:
Might be a good idea to provide more in depth information about semantic versioning for people who aren't familiar. You could do something similar to: https://eips.ethereum.org/EIPS/eip-5139#versioning